src/style.css

```
@@ -1,3 +1,74 @@
 body {
   margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+  --blue-color: #00acdc;
+  --main-color: #000;
+}
+
+.header {
+  background: #fff;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.nav__link {
+  text-decoration: none;
+  display: block;
+  padding: 22.8px 0;
+  color: var(--main-color);
+}
+
+.nav__list {
+  text-transform: uppercase;
+  list-style: none;
+  display: flex;
+  flex-direction: row;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
+  margin: 0 10px;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+  flex-shrink: 0;
+  margin: 10px 0;
+}
+
+.nav__list :hover {
+  color: var(--blue-color);
+}
+
+a.is-active {
+  position: relative;
+  color: var(--blue-color);
+  width: 100%;
+}
+
+a.is-active::after {
```

```suggestion
.nav__link.is-active::after {
```
It's better to use a class selector instead of a tag for some specific styles added just for navigation